docs/user-guide/scheduled-jobs.md

```diff
@@ -0,0 +1,177 @@
+delete all jobs in the system (not just the ones created by "hello".)
+
+## Scheduled Job Limitations
+
+A scheduled job creates a job object _about_ once per execution time of its schedule. We say "about" because there
+are certain circumstances where two jobs might be created, or no job might be created. We attempt to make these rare,
+but do not completely prevent them. Therefore, jobs should be _idempotent_.
+
+The job is responsible for retrying pods, parallelism among pods it creates, and determining the success or failure
+of the set of pods. A scheduled job does not examine pods at all.
+
+## Writing a Scheduled Job Spec
+
+As with all other Kubernetes configs, a scheduled job needs `apiVersion`, `kind`, and `metadata` fields. For general
+information about working with config files, see [deploying applications](/docs/user-guide/deploying-applications), 
+[configuring containers](/docs/user-guide/configuring-containers), and
+[using kubectl to manage resources](/docs/user-guide/working-with-resources) documents.
+
+A scheduled job also needs a [`.spec` section](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/api-conventions.md#spec-and-status).
+
+### Schedule 
+
+The `.spec.schedule` is a required field of the `.spec`. It takes a [Cron](https://en.wikipedia.org/wiki/Cron) format
+string, e.g. `0 * * * *` or `@hourly`, as schedule time of its jobs to be created and executed. 
+
+### Job Template
+
+The `.spec.jobTemplate` is another required field of the `.spec`. It is a job template. It has exactly the same schema
+as a [Job](/docs/user-guide/jobs), except it is nested and does not have an `apiVersion` or `kind`, see
+[Writing a Job Spec](/docs/user-guide/jobs/#writing-a-job-spec).
+
+### Starting Deadline Seconds
+
+The `.spec.startingDeadlineSeconds` field is optional. It stands for the deadline (in seconds) for starting the job
+if it misses its scheduled time for any reason. Missed jobs executions will be counted as failed ones. If not specified, 
+there's no deadline.  
+
+### Concurrency Policy
+
+The `.spec.concurrencyPolicy` field is also optional. It specifies how to treat concurrent executions of a job
+created by this scheduled job. Only one of the following concurrent policies may be specified:
+
+* `Allow` (default): allows concurrently running jobs
+* `Forbid`: forbids concurrent runs, skipping next run if previous hasn't finished yet
+* `Replace`: cancels currently running job and replaces it with a new one
+
+Note that concurrency policy only applies to the jobs created by the same scheduled job. If there are multiple
+scheduled jobs, their respective jobs are always allowed to run concurrently.
+
+### Suspend
+
+The `.spec.suspend` field is also optional. If set to `true`, all subsequent executions will be suspended. It does not 
+apply to already started executions. Defaults to false. 
```

Comment:
I'd add a not that all modifications to SJ (esp. the `.spec`) part are applied only to the next run.